parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -61,7 +61,10 @@ public class ParselyTracker {
     private static final String STORAGE_KEY = "parsely-events.ser";
 // emulator localhost
 //    private static final String ROOT_URL = "http://10.0.2.2:5001/";
-    private static final String ROOT_URL = "https://p1.parsely.com/";
+    /**
+     * @noinspection StringOperationCanBeSimplified
+     */
+    private static final String ROOT_URL = "https://p1.parsely.com/".intern();
```

Comment:
1. Praise (❤️): TIL about that and `intern()`, never seen that before (ed56a27b99ef0c45cdcd466516f089b780cf910a)! 😮
2. Question (❓): I wonder whether we can do something better about this `ROOT_URL`, it feels to be better suited for a configuration file and not something that should exist as a private static final string... 🤷 Although, I do understand that configuring this should be possible as we always want to point to this url, as such I am a bit torn here... 🤔
3. Suggestion (💡): I suggest using `@SuppressWarnings("StringOperationCanBeSimplified")` instead of suppressing that via a Javadoc, just like we do elsewhere.
4. Minor (🔍): I suggest adding `.intern()` on the emulator localhost related `ROOT_URL` as well, just to avoid any question on why one is with and why the other without... 🤷 
5. Suggestion (💡): Also, as I recommended before, I again suggest refactoring this production `ROOT_URL` and its emulator equivalent to become side-by-side so that when commenting that out-and-in it will become as effortless as possible, with the suppress warning still being applicable to both before-and-after the change:

`ROOT_URL` production:

![image](https://github.com/Parsely/parsely-android/assets/9729923/991e98c6-8ea6-4a34-974b-e0c8ef97140c)

`ROOT_URL` emulator localhost:

![image](https://github.com/Parsely/parsely-android/assets/9729923/dea06686-ed31-4f81-a704-e8f0f239b0f7)

Reply:
5dcb5ffb5e8db8a8fe373c7874816529f4b44b9e + a266ab00eacdb003f6d282bbc80b294de556c797 is looking good, thanks! 👍